.github/actions/test/action.yml

```diff
@@ -1,5 +1,10 @@
 name: Test Python
 description: Runs Python tests and checks coverage when tests pass
+inputs:
```

Comment:
Shouldn't this be required? Or is it not because of the default?

Reply:
I've not marked it required because there is a default.